apps/bmd/src/pages/PollWorkerScreen.tsx

```diff
@@ -182,6 +219,81 @@ const PollWorkerScreen: React.FC<Props> = ({
     )
   }
 
+  const isMachineTallySaved =
+    talliesOnCard &&
+    !!talliesOnCard.metadata.find(
+      (metadata) => metadata.machineId === machineConfig.machineId
+    )
+
+  const combinedTally =
+    talliesOnCard === undefined
+      ? tally
+      : isMachineTallySaved
+      ? talliesOnCard.tally
+      : combineTallies(election, talliesOnCard.tally, tally)
+
+  const combinedBallotsPrinted =
+    talliesOnCard === undefined
+      ? ballotsPrintedCount
+      : isMachineTallySaved
+      ? talliesOnCard.totalBallotsPrinted
+      : talliesOnCard.totalBallotsPrinted + ballotsPrintedCount
+
+  const handleSavingTally = async () => {
+    setIsSavingTally(true)
+    const metadata = talliesOnCard?.metadata ?? []
+    metadata.push({
+      machineId: machineConfig.machineId,
+      timeSaved: Date.now(),
+    })
+    await saveTallyToCard({
+      tally: combinedTally,
+      metadata,
+      totalBallotsPrinted: combinedBallotsPrinted,
+    })
+    setIsSavingTally(false)
+  }
+  const metadataRows = []
```

Comment:
Give this a type so it isn't inferred as `any[]`.
```suggestion
  const metadataRows: React.ReactChild[] = []
```